airflow/jobs.py

```diff
@@ -398,6 +403,7 @@ def schedule_dag(self, dag):
                 )
                 session.add(next_run)
                 session.commit()
```

Comment:
is there the need to expunge_all after the commit? I think otherwise you'll get errors when reading properties of the object.